src/pages/LEVANTE/RegisterUsers.vue

```diff
@@ -155,6 +155,34 @@ const allFields = [
     dataType: 'string',
   },
 ];
+// Used to remove all other fields in the uploaded CSV
+const allFieldsIdsList = allFields.map((a) => a.field);
+const sanitizeAndValidateSubmittedUsers = (users) => {
+  const userIdMap = {};
+  const filteredFieldsUsers = users.map((user) => {
+    const acc = {};
+    Object.keys(user).forEach((uKey) => {
+      if (allFieldsIdsList.includes(uKey)) {
+        acc[uKey] = user[uKey];
+      }
+    });
+    // Allowing 0 as a valid id
+    const idList = [user['childId'], user['teacherId'], user['parentId']].filter((a) => 0 || !!a);
+    if (idList.length > 1) {
+      addErrorUser(user, `Multiple ids present for the user: ${user.id}`);
+      throw new Error(`Mutliple ids found for user, please see the table below for more details`);
+    }
+    if (userIdMap[idList[0]]) {
+      addErrorUser(user, `Duplicate id present with id: ${idList[0]}`);
+      throw new Error(`Duplicate ids found for user, please see the table below for more details`);
```

Comment:
There seems to be a problem with this logic. If I try to register only child users this error is thrown.

Input:
![Screenshot 2024-06-06 at 12 28 00 PM](https://github.com/yeatmanlab/roar-dashboard/assets/80180474/b7bba6e8-83ca-48ef-861f-8da3a3291cde)

Error:
![Screenshot 2024-06-06 at 12 28 42 PM](https://github.com/yeatmanlab/roar-dashboard/assets/80180474/c7d59880-46e6-42e1-acad-4fe62bb4d1d8)